addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -314,14 +323,18 @@ odoo.define('website.tour.form_editor', function (require) {
         {
             content:  "Wait reloading...",
             trigger:  "html:not(:has(#completlyloaded)) div",
+        },
+        {
+            content: 'Verify value attribute and property',
+            trigger: '.s_website_form_field:eq(0) input[value="John Smith"]:propValue("Mitchell Admin")',
```

Comment:
It's cool to have this test about default value but does this cover the bug you are fixing in this PR? From what I understand the bug occurs on re-edition of the form which already has a default value, this test does not test that, right? :thinking:

Reply:
Well in fact the commit should indeed be split, I was going to accept that it's not but it really is confusing. So yes split the fix and the auto-fill priority in two separate commits. Actually not sure if it should be for stable or not, but in this case, I would keep it for stable. So split in tho commits, in this PR.

Still, I think the test should be further improved to test the bug you fix here (by reediting the form that has a default value).